### ISSUE TITLE
Wrong code (L884 for Vignola [TN]) used instead of L885 when encoding a person born in Vignola (MO) 

### ISSUE THREAD

**Issue by giobber** [fabiocaccamo/python-codicefiscale#37]
**Python version**
3.10

**Package version**
Installing `main` repository with `pip` commit: 3b78e0db8fc83f2d48384e1152950f9dd8f3ac2e

**Current behavior (bug description)**
When encoding a CF for a person born in Vignola (MO) `.encode` uses code L884 (for Vignola (TN) between 1920-1929)  instead of L885.

**Expected behavior**
Use the right code L885 for a person born in Vignola (MO) after 1861

**Comment by giobber** [comment]
Moving block for L885 in `municipalities.json` after the two block for code L884 seem to solve

```json
    ...
    {
        "active": false,
        "code": "L884",
        "date_created": "1920-10-16",
        "date_deleted": "1923-02-13",
        "name": "Vignola",
        "name_alt": "",
        "name_alt_trans": "",
        "name_slugs": [
            "vignola"
        ],
        "name_trans": "Vignola",
        "province": "TN"
    },
    {
        "active": false,
        "code": "L884",
        "date_created": "1923-02-14",
        "date_deleted": "1929-01-24",
        "name": "Vignola",
        "name_alt": "",
        "name_alt_trans": "",
        "name_slugs": [
            "vignola"
        ],
        "name_trans": "Vignola",
        "province": "TN"
    },
    {
        "active": true,
        "code": "L885",
        "date_created": "1861-03-17",
        "date_deleted": "",
        "name": "Vignola",
        "name_alt": "",
        "name_alt_trans": "",
        "name_slugs": [
            "vignola"
        ],
        "name_trans": "Vignola",
        "province": "MO"
    },
    ...
```

**Comment by giobber** [comment]
I found the problem, if I pass as birthplace "Vignola (MO)" it work correctly, if I pass "Vignola" it doesn't.
In file [codicefiscale.py](https://github.com/fabiocaccamo/python-codicefiscale/blob/master/codicefiscale/codicefiscale.py) at row 95-97 
```python
        for name in names:
            data["municipalities"][name] = municipality
            data["municipalities"][name + "-" + province] = municipality
```
first assignment get the same slug/name "vignola" for either Vignola (MO) and Vignola (TN), so the expected Vignola (MO) when searched with slug "vignola" is overwritten by the last Vignola (TN)
Now, for what reason the birthdate check in row 426-427 is not working for Vignola (TN) is a mistery

**Comment by fabiocaccamo** [comment]
@giobber thank you for reporting this.

**Comment by fabiocaccamo** [comment]
@giobber this is fixed in `0.6.0` version.
